checkpoints/fourth.patch

```diff
@@ -0,0 +1,95 @@
+diff --git a/frontend/src/components/Tweet/ComposeTweet.jsx b/frontend/src/components/Tweet/ComposeTweet.jsx
+index 2030936..393d865 100644
+--- a/frontend/src/components/Tweet/ComposeTweet.jsx
++++ b/frontend/src/components/Tweet/ComposeTweet.jsx
+@@ -19,9 +19,7 @@ export default function ComposeTweet({ handle }) {
+   const onTextAreaChange = (e) => {
+     const input = e.target.value;
+ 
+-    if (input.length > TEXT_CHAR_LIMIT) {
+-      return;
+-    }
++    // Task 2D: Restrict the end user from writing any more characters when they reach the maximum number of characters allowed.
+ 
+     setTextArea(input);
+   };
+@@ -31,20 +29,14 @@ export default function ComposeTweet({ handle }) {
+   }, []);
+ 
+   const getCharLimitIndicatorColor = () => {
+-    if (textArea.length > 200 && textArea.length < TEXT_CHAR_LIMIT) {
+-      return "text-yellow-600";
+-    }
+-
+-    if (textArea.length >= TEXT_CHAR_LIMIT) {
+-      return "text-red-600";
+-    }
++    // Task 2C : Make charachter limit indicator to red when user reaches the limit?
+   };
+ 
+   const onTweetSubmit = (e) => {
+     e.preventDefault();
+ 
+     fetch(tweetRoute, {
+-      method: "POST",
++      // Task 3A : What HTTP request method do you think is most appropriate to use here
+       credentials: "include",
+       headers: {
+         "Content-Type": "application/json",
+@@ -56,17 +48,11 @@ export default function ComposeTweet({ handle }) {
+       .then((response) => {
+         if (response.ok) {
+           response.json().then((body) => {
+-            // @TODO: Redirect the user to the single tweet view
+-            router.push(`/${user.handle}/status/${body.data.tweet.id}`);
++            // Task 3B - Redirect the user to the single tweet view
+           });
+         } else {
+-          // If create tweet fails
+           response.json().then((body) => {
+-            if (body?.message) {
+-              toast.error(body.message);
+-            } else {
+-              toast.error(uiText.somethingWentWrong);
```

Comment:
Keep the if/else conditions visible. Just keep the toast.error portion a todo

Reply:
Ack